pkg/pillar/types/attesttypes.go

```diff
@@ -21,29 +21,13 @@ func (nonce AttestNonce) Key() string {
 //SigAlg denotes the Signature algorithm in use e.g. ECDSA, RSASSA
 type SigAlg uint8
 
-//CertType carries the certificate use case e.g. ek, ecdh_exchange etc
-type CertType uint8
-
-//CertHashType carries the hash algo used for compute the short hash
-type CertHashType uint8
-
```

Comment:
Why do you have to move this? It is still in the same package hence just noise to move it.